question-servers/freeform.js

```diff
@@ -56,6 +57,13 @@ module.exports = {
             return '';
         } else if (phase == 'file') {
             return '';
+        } else if (phase == 'get_dependencies') {
```

Comment:
We've been avoiding the `get_` prefix on element function names. Should this be just `dependencies()`?